keras/backend/tensorflow_backend.py

```diff
@@ -773,18 +813,54 @@ def cast(x, dtype):
 
 
 def update(x, new_x):
+    """Update the value of 'x' to 'new_x'.
```

Comment:
Code delimiters ` would be more appropriate than string quotes here, for `x` and `new_x`.